protocol/l2-blob.md

```diff
@@ -0,0 +1,199 @@
+# Purpose
+
+<!-- This section is also sometimes called “Motivations” or “Goals”. -->
+
+<!-- It is fine to remove this section from the final document,
+but understanding the purpose of the doc when writing is very helpful. -->
+Currently, EIP-4844 BLOB transactions are disabled on the OP Stack because Layer 2 (L2) BLOBs offer no advantage over L2 calldata when both use the same L1 DA. This forces L3s to rely on the same DA as their L2, unless they want to integrate third-party DA providers or manage the security risks associated with DA bridges. However, different DA solutions have their own advantages—L1 DA provides better security, while [Alt-DA](https://github.com/ethereum-optimism/specs/blob/main/specs/experimental/alt-da.md) offers lower costs.
```

Comment:
> This forces L3s to rely on the same DA as their L2

This sentence is a bit confusing for me, it reads like this spec makes L3s be able to use a DA not provided by L2. It's a fact that L3s can only use DA provided by L2, or say L3s can not choose to use a DA out of the box.